make/modules/jdk.hotspot.agent/Lib.gmk

```diff
@@ -31,7 +31,7 @@ ifeq ($(call isTargetOs, linux), true)
   SA_CFLAGS := -D_FILE_OFFSET_BITS=64
 
 else ifeq ($(call isTargetOs, macosx), true)
-  SA_CFLAGS := -Damd64 -D_GNU_SOURCE -mno-omit-leaf-frame-pointer \
```

Comment:
this and other SA fixes
we have same but not shared it yet because it discloses information available only in darwin_aarch64 part of macos11.0sdk. it can't be shared under xcode12 beta license agreement